.github/CODEOWNERS

```diff
@@ -0,0 +1,76 @@
+# Describes which individuals/teams are reponsible for which code in the
+# repository.
+#
+# See: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/.cargo                             @benesch
+/.config                            @benesch
+/.devcontainer                      @mjibson
+/.github                            @benesch
+/bin
+/ci                                 @benesch
```

Comment:
This is here twice, and I think the latter binding is more appropriate.

Reply:
Oops, thanks!